src/components/CreatePostForm/CreatePostForm.scss

```diff
@@ -0,0 +1,12 @@
+@use "../../styles/mixin";
+
+.form__container {
+  background-color: #b0ddc98a;
+  max-width: 450px;
+  max-width: 400px;
```

Comment:
max-width 450 o 400? 
![KetchupSimpsonsGIF](https://github.com/MarcosRodriguezAlonso/the-why-blog/assets/128187875/df82edce-7dae-47ef-8765-5bcd09ebe27b)

Reply:
Juraría que puse max-height pero obviamente no

Reply:
Ya está arreglado.